.github/workflows/test.yml

```
@@ -73,6 +59,60 @@ jobs:
           PUBLIC_PINATA_GATEWAY_URL: ${{ secrets.PUBLIC_PINATA_GATEWAY_URL }}
           PRIVATE_PINATA_GATEWAY_KEY: ${{ secrets.PRIVATE_PINATA_GATEWAY_KEY }}
 
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)